pom.xml

```diff
@@ -30,13 +30,13 @@
     <surefire.redirectTestOutputToFile>true</surefire.redirectTestOutputToFile>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <!-- version properties -->
-    <cdap.version>6.2.0-SNAPSHOT</cdap.version>
+    <cdap.version>6.2.3</cdap.version>
```

Comment:
Is this plugin only compatible with CDAP 6.2.3 and later versions? If not, I would just remove the -SNAPSHOT suffix. Same thing for hydrator.version.

Reply:
The plugin is compatible with 6.1, 6.2 and 6.3. Let me know what do you want to me to set as default in the pom.xml.

Reply:
It was 6.2.0-SNAPSHOT before your change, so let's set it to 6.2.0

Reply:
Done.